pom.xml

```diff
@@ -223,6 +217,18 @@
     </dependency>
 
     <dependency>
+      <groupId>org.slf4j</groupId>
+      <artifactId>slf4j-log4j12</artifactId>
+      <version>1.7.24</version>
+    </dependency>
+
+    <dependency>
+      <groupId>log4j</groupId>
+      <artifactId>log4j</artifactId>
+      <version>1.2.17</version>
```

Comment:
Is Spark stuck on log4j 1? Will log4j2 not work with it? I just ask because log4j 1 is EOL now (last release being 5 years ago).